stripe/src/main/java/com/stripe/android/auth/PaymentAuthWebViewContract.kt

```diff
@@ -16,15 +17,28 @@ import kotlinx.parcelize.Parcelize
  * TODO(mshafrir-stripe): use a more generic class name
  */
 internal class PaymentAuthWebViewContract(
-    private val defaultReturnUrl: DefaultReturnUrl
+    private val defaultReturnUrl: DefaultReturnUrl,
+    private val isCustomTabsSupported: (Context) -> Boolean
```

Comment:
can we just add `{ context -> CustomTabsCapabilities(context).isSupported() }` here to avoid declaring 
 the other constructor?

Reply:
Sure